include/z_en_hy_code.h

```diff
@@ -36,6 +36,25 @@ typedef enum {
 
 typedef void (*EnHyActionFunc)(struct EnHy*, PlayState*);
 
+typedef enum EnHyBodyPart {
+    /*  0 */ ENHY_SHADOW_BODYPART_0,
```

Comment:
Ah right. I basically added the extra shadow qualifier because of `EnGrasshopper` (dragonfly). That was a case where that actor had two sets of bodyParts, 1 for the shadow via `SubS_GenShadowTex` and 1 for the effects (`Actor_SpawnIceEffects` and `Actor_DrawDamageEffects`). So to distinguish them, I made two bodyPart enums and I added the `SHADOW` qualifier to the enum for the shadow body parts.

Now, `SubS_GenShadowTex` is only used 5 times, 2 of which use the `EnHy` values, so I figured I'd add the `SHADOW` qualifier to those just for consistency. The `EnHyBodyPart`, `gEnHyLimbToShadowBodyParts`, and `gEnHyParentShadowBodyParts` are only ever used for `SubS_GenShadowTex` so I figured it'd be fine. One could also add the `EFFECTS` qualifier to most body parts since the most common use-case of these are the two `Actor_SpawnIceEffects` and `Actor_DrawDamageEffects`, but idk.

That's my motivation anyway, thoughts?

Reply:
Ah that is annoying. I'm not 100% sure, but I wonder if it would be better for everything to not have qualifiers, except `Grasshopper` which adds qualifiers to both 'shadowBodyPart` and `effectBodyPart` (there is also the case of the `Ik` which has two different body parts, so one/both need a qualifier at some point)?

Reply:
Yeah, that's fair. I can drop the SHADOW except for the grasshopper. There's a couple others that have have two sets of body parts, but without documented limbs, I just kept it as `sLimbToBodyParts1` and `sLimbToBodyParts2`

Reply:
Ended up mostly dropping the Shadow, but left it on `sParentShadowBodyParts` since those parent-structure body parts are really only used for the shadows?